plugin/keyring_vault/vault_base64.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MYSQL_VAULT_BASE64_H
+#define MYSQL_VAULT_BASE64_H
+
+#include <my_global.h>
+#include "vault_memory.h"
+
+namespace keyring
+{
+  class Vault_base64
```

Comment:
There are only functions in Vault_base64, for me struct is more like container of simple types.